### DISCUSSION TITLE
Are there any security issues in sending json data, b/w .NET app and React app running in Cef browser, through Cef events?

### DISCUSSION THREAD

**Discussion by Ayyappu** [cefsharp/CefSharp#3563] [Q&A]
I have a .NET (WinForms) app. A React app can be launched from inside the WinForms app (this React app is loaded in Cef browser). I am able to send data between the 2 apps (WinForms and React inside Cef) using [ExecuteJavaScriptAsync method](https://cefsharp.github.io/api/63.0.0/html/M_CefSharp_IFrame_ExecuteJavaScriptAsync.htm). I noticed that I am able to send json data of any size i.e., there are no size limitations. Is sending such a big json data via Cef events safe? Can the confidential json data be hacked by anyone?

**Comment by amaitland** [comment]
Already asked at https://stackoverflow.com/questions/67568072/are-there-any-security-issues-in-sending-json-data-b-w-net-app-and-react-app-r continue the conversation there.

In future please ask your question in a single place, either here or GitHub, not both. Thanks.
